airbyte-integrations/connectors/source-datadog/unit_tests/conftest.py

```diff
@@ -8,6 +8,7 @@
 @fixture(name="config")
 def config_fixture():
     return {
+        "site": "api.datadoghq.com",
```

Comment:
`test_site_parameter_is_not_set` is failing:

```
                 expected_url_base = "https://api.datadoghq.com/api"
         >       assert url_base == expected_url_base
         E       AssertionError: assert 'https://api.api.datadoghq.com/api' == 'https://api.datadoghq.com/api'
         E         - https://api.datadoghq.com/api
         E         + https://api.api.datadoghq.com/api
         E         ?         ++++

         unit_tests/test_streams.py:130: AssertionError
```

This fixes it:
```suggestion
        "site": "datadoghq.com",
```